.github/workflows/publish_internal.yaml

```diff
@@ -42,11 +43,36 @@ jobs:
           PR_LABELS: "${{ join(github.event.pull_request.labels.*.name) }}"
         run: dart pkgs/firehose/bin/firehose.dart --validate ${{ fromJSON('{"true":"--use-flutter","false":"--no-use-flutter"}')[env.use-flutter] }}
 
+      - name: Get comment id
+        if: ${{ (hashFiles('output/comment.md') != '') && fromJSON(env.write-comments ) }}
+        run: |
+          touch -a output/commentId
+          COMMENT_ID=$(cat output/commentId)
+          echo "COMMENT_ID=$COMMENT_ID" >> $GITHUB_ENV
+
+      - name: Create comment
+        uses: peter-evans/create-or-update-comment@46da6c0d98504aed6fc429519a258b951f23f474
+        if: ${{ (hashFiles('output/comment.md') != '') && fromJSON(env.write-comments ) && (env.COMMENT_ID == '') }}
```

Comment:
`fromJSON` is needed when using `env` variables, as they are always strings, while `inputs` can be boolean. So `fromJSON(env.write-comments ) == false` and `env.write-comments  == 'false'` while `inputs.write-comments  == false`. Of course `false != 'false'`...

fyi @HosseinYousefi